files/fr/web/css/css_selectors/index.md

```diff
@@ -52,7 +52,7 @@ original_slug: Web/CSS/Sélecteurs_CSS
     **Syntaxe :** `A > B`
     **Exemple :** `ul > li` permettra de cibler tous les éléments {{HTMLElement('li')}} qui sont directement situés sous un élément {{HTMLElement('ul')}}.
 - [Les sélecteurs d'éléments descendants](/fr/docs/Web/CSS/Descendant_combinator)
-  - : Le combinateur ` ` (espace) permet de sélectionner les nœuds qui sont des descendants (pas nécessairement des enfants directs) d'un élément donné.
+  - : Le combinateur `&nbsp;` (espace) permet de sélectionner les nœuds qui sont des descendants (pas nécessairement des enfants directs) d'un élément donné.
```

Comment:
This didn't work

Reply:
Switched to quotes to match upstream, but didn't use the french style quotemarks